docs/hpc/03_storage/09_transferring_cloud_storage_data_with_rclone.md

```diff
@@ -47,7 +47,7 @@ This will try to open the config files and you will see the below content:
 
 You can select one of the options (here we show how to set up a new remote)
 ```sh
-2025/03/14 16:39:23 NOTICE: Config file "/Users/rjy1/.config/rclone/rclone.conf" not found - using defaults
```

Comment:
Easier to switch to either `$USER` or `$NetID` to prevent confusion among users who might copy this directly.

Reply:
Good idea.  I'll change it to $USER.  Thanks!